topics/ecology/tutorials/champs-blocs/tutorial_fr.md

```diff
@@ -0,0 +1,199 @@
+---
+layout: tutorial_hands_on
+
+title: Production d'indicateurs champs de bloc
+questions:
+- Which biological questions are addressed by the tutorial?
+- Which bioinformatics techniques are important to know for this type of data?
+objectives:
+- Calcul de l'indice IVR (Visual Rollover Indicator)
+- Calcul de l'indice de dissimilarité
+- Visualisation de quelques indices de diversité
+time_estimation: 1H
+key_points:
+- Indicateurs de diversité
+- Champs de Blocs
+contributors:
+- Marie59
+- yvanlebras
+
```

Comment:
please add the following to the metadata:

```
lang: fr
translations:
    - en
 ```
 
 and then to the english version of the tutorial:
 
 ```
 translations:
    - fr
 ```